vsphere/data_source_vsphere_entity_permission.go

```diff
@@ -0,0 +1,54 @@
+package vsphere
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	permissionsHelper "github.com/terraform-providers/terraform-provider-vsphere/vsphere/internal/helper/permissions"
```

Comment:
This is very nit-picky and there may be a reason you've got it this way, but just to keep consistent with the other resources, I'd like to avoid using an alias unless necessary.